Dockerfile

```diff
@@ -25,6 +25,21 @@ COPY Makefile Makefile
 COPY database/sqlite database/sqlite
 RUN make sqlite-db-refresh
 
+# INSTALL POSTGRESQL
+RUN wget https://ftp.postgresql.org/pub/source/v14.0/postgresql-14.0.tar.gz \
+    && tar -zxf postgresql-14.0.tar.gz \
+    && cd postgresql-14.0 \
+    && ./configure --without-readline --without-zlib\
+    && make \
+    && make install 
+
+# USER 'make' and 'psql' to create the dev database
+COPY Makefile Makefile
+COPY database/postgresql database/psql
+# TODO: psql not found error when running psql commands (uncomment below)
```

Comment:
Add creation of PostgreSQL database, tables, and populate tables.

Reply:
These steps are in `RUN make psql-db-refresh`  line, but because I'm encountering a "psql not found" error even after adding psql to the Docker PATH, Jimmy said to just leave the ticket as is here.